cocoa/tests_backend/screen.py

```diff
@@ -0,0 +1,23 @@
+from toga_cocoa.libs import NSScreen
+
+from .probe import BaseProbe
+
+
+class ScreenProbe(BaseProbe):
+    def __init__(self, screen):
+        super().__init__()
+        self.screen = screen
+        self._impl = screen._impl
+        self.native = screen._impl.native
+        assert isinstance(self.native, NSScreen)
+
+    def assert_name(self):
+        assert self.screen.name == self.native.localizedName
+
+    def assert_origin(self):
+        frame_native = self.native.frame
+        assert self.screen.origin == (frame_native.origin.x, frame_native.origin.y)
```

Comment:
Again - you *don't* need to have the *assertion* in the probe. The *assertion* is the same on every platform - what is different is the way you obtain the raw value. Therefore the *assertion* should be in the test, and the *probe* should contain the mechanism for retrieving the data that you're asserting.

Reply:
Removed it.